<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distractionFree</class>
 <widget class="QDialog" name="distractionFree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2"/>
 </widget>
 <resources/>
 <connections/>
</ui>
